packages/orbit-components/src/Popover/components/ContentWrapper.tsx

```diff
@@ -166,7 +166,7 @@ const PopoverContentWrapper = ({
           "h-auto w-full",
           "z-[1000]",
           "box-border",
-          "shadow-raised-reverse",
+          "shadow-level3-reverse",
           "bg-elevation-raised-reverse",
```

Comment:
How are we addressing these? This shadow will disappear but the bg-color remains? Should we deprecate it as well and just use the color directly? In this case, it would be just bg-white-normal, no?

Reply:
Yeah, backgrounds are still missing.. I need to tackle this and probably just use the replacement (white normal as you suggested).

Reply:
I see you kept the tokens or renamed them accordingly. I'm ok with that, but I had suggested to completely get rid of those tokens and use the palette directly.

In this case we would use `bg-white-normal` instead of `bg-elevation-raised-reverse` that has the value of white.

And btw, raised reverse is deprecated, so this needs to be updated to `bg-level3-reverse`, right?

Reply:
I'm removing tokens and using `bg-white-normal`